flink-formats/flink-avro-confluent-registry/src/main/java/org/apache/flink/formats/avro/registry/confluent/ConfluentRegistryAvroDeserializationSchema.java

```diff
@@ -84,10 +86,47 @@ private ConfluentRegistryAvroDeserializationSchema(
      */
     public static ConfluentRegistryAvroDeserializationSchema<GenericRecord> forGeneric(
             Schema schema, String url, int identityMapCapacity) {
+        return forGeneric(schema, url, identityMapCapacity, null);
+    }
+
+    /**
+     * Creates {@link ConfluentRegistryAvroDeserializationSchema} that produces {@link
+     * GenericRecord} using provided reader schema and looks up writer schema in Confluent Schema
+     * Registry.
+     *
+     * @param schema schema of produced records
+     * @param url url of schema registry to connect
+     * @param registryConfigs map with additional schema registry configs (for example ssl
+     *     properties: 'schema.registry.ssl.keystore.location',
```

Comment:
avoid code duplication and simplify `map with additional schema registry configs (for example for SSL properties).`